web-components/src/components/code-editor/CodeEditor.stories.ts

```diff
@@ -0,0 +1,21 @@
+import "./CodeEditor";
+import { withA11y } from "@storybook/addon-a11y";
+import { html } from "lit-element";
+import { withKnobs } from "@storybook/addon-knobs";
+
+export default {
+  title: "Code Editor",
+  component: "md-code-editor",
+  decorators: [withKnobs, withA11y],
+  parameters: {
+    a11y: {
+      element: "md-code-editor"
+    }
+  }
+};
+
+export const CodeEditor = () => {
+  return html`
+    <md-code-editor></md-code-editor>
```

Comment:
I would add the properties as knobs to this storybook component:
acceptLanguage, getLocalization, copyLocalization, copiedLocalization

Then run `yarn storybook` locally to test codeEditor in storybook

Reply:
I added only acceptLanguage prop. the others are not needed for storybook